ts/features/it-wallet/components/ClaimsList.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { List } from "native-base";
```

Comment:
Is it right to use List from native-base?

Reply:
Nope, nice catch. Removed it.